index.bs

```diff
@@ -2194,6 +2197,75 @@ To <dfn>get the navigation info</dfn>, given |context| and |navigation status|:
 
 ### Commands ### {#module-browsingContext-commands}
 
+#### The browsingContext.create Command ####  {#command-browsingContext-create}
+
+The <dfn export for=commands>browsingContext.create</dfn> command creates a new
+[=/browsing context=], either in a new window or in a new tab, and returns its
```

Comment:
Given that the tab is the default now, maybe we should switch the order?

Reply:
There is no default, but I'll change the order anyway to match the alphabetical order below.